app-admin/graylog/graylog-6.0.5.ebuild

```diff
@@ -1,24 +1,24 @@
-# Copyright 1999-2022 Gentoo Authors
+# Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
 DESCRIPTION="Free and open source log management"
-HOMEPAGE="https://www.graylog.org"
+HOMEPAGE="https://graylog.org"
 SRC_URI="https://downloads.graylog.org/releases/graylog/${P}.tgz"
 
 LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="~amd64 ~ppc64 ~x86"
 RESTRICT="strip"
 
-RDEPEND="acct-group/graylog
+RDEPEND="
+	acct-group/graylog
 	acct-user/graylog
-	>=virtual/jdk-1.8:*"
+	>=virtual/jdk-1.8:*
+"
 
-DOCS=(
-	README.markdown UPGRADING.rst
-)
+DOCS=( AUTHORS README.markdown SECURITY.md UPGRADING.md )
```

Comment:
installation phase dies with following message:
```
>>> Install app-admin/graylog-6.0.5 into /var/tmp/portage/app-admin/graylog-6.0.5/image
Traceback (most recent call last):
  File "/usr/lib/portage/python3.13/doins.py", line 609, in <module>
    sys.exit(main(sys.argv[1:]))
             ~~~~^^^^^^^^^^^^^^
  File "/usr/lib/portage/python3.13/doins.py", line 598, in main
    if _doins(
       ~~~~~~^
        opts, install_runner, os.path.basename(source), os.path.dirname(source)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ):
    ^
  File "/usr/lib/portage/python3.13/doins.py", line 441, in _doins
    return install_runner.install_file(source, os.path.dirname(dest))
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/portage/python3.13/doins.py", line 375, in install_file
    return self._ins_runner.run(source, dest_dir)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/usr/lib/portage/python3.13/doins.py", line 183, in run
    sstat = os.stat(source)
FileNotFoundError: [Errno 2] No such file or directory: b'AUTHORS'
 * ERROR: app-admin/graylog-6.0.5::gentoo failed (install phase):
 *   dodoc failed
```
I see only `LICENSE` and   `README.markdown` in `${S}` from all listed files here